js/activity.js

```diff
@@ -3098,9 +3098,11 @@ function Activity() {
                     [jsONON.length,"hidden",0,0,[val, last ? null: jsONON.length +1]]                  
                 );
             }
-            console.debug(JSON.stringify(jsONON));
-            console.debug ('midi track loading ... be patient ');
+            //console.debug(JSON.stringify(jsONON));
             console.debug ('finished when you see: "block loading finished "');
+            document.body.style.cursor = "wait";
```

Comment:
do you reset the cursor after the load completes?

Reply:
yes , there was a reset present in blocks.js L6839